rss/serializers/atom_extensions.py

```diff
@@ -0,0 +1,63 @@
+from feedgen.ext.base import BaseExtension, BaseEntryExtension
+from lxml import etree
+
+
+class ArxivExtension(BaseExtension):
+    def __init__(self):
+        pass
+
+    def extend_atom(self, atom_feed):
+        return atom_feed
+
+    def extend_rss(self, rss_feed):
+        return rss_feed
+
+    def extend_ns(self):
+        return {'arxiv': 'http://arxiv.org/schemas/atom'}
+
+
+class ArxivEntryExtension(BaseEntryExtension):
+
+    def __init__(self):
+
+        self.__arxiv_comment = None
+        self.__arxiv_primary_category = None
+        self.__arxiv_doi = None
+        self.__arxiv_affiliation = None
+        self.__arxiv_journal_ref = None
+
+    def extend_atom(self, entry):
+
+        if self.__arxiv_comment:
+            comment_element = etree.SubElement(entry, '{http://arxiv.org/schemas/atom}comment')
```

Comment:
Just curious, is this syntax: `'{http://arxiv.org/schemas/atom}comment'` something specific to lxml? Looks sort of like an xpath, but  with a URI prefix in braces.